Odata-docs/client/bulk-update-operations.md

```diff
@@ -0,0 +1,284 @@
+
+DataServiceResponse response = await context.BulkUpdateAsync(customer);
+
+// You can traverse the DataServiceResponse object like below: 
+var changeOperationResponse = response.First() as ChangeOperationResponse;
+EntityDescriptor entityDescriptor = changeOperationResponse.Descriptor as EntityDescriptor;
+var returnedCustomer = entityDescriptor.Entity as Customer;
+var nestedResponse = changeOperationResponse.NestedResponses[0] as ChangeOperationResponse;
+LinkDescriptor orderDescriptor = nestedResponse.Descriptor as LinkDescriptor;
+var linkedOrder = orderDescriptor.Target as Order;
+
+Assert.Equal("Customer1", returnedCustomer.Name);
+Assert.Equal(1, linkedOrder.Id);
+```
+The code above generates the following bulk update request payload: 
+
+```http
+PATCH http://localhost:6285/odata/Customers
+```
+Request body:
+
+```json
+{
+  "@context": "http://localhost:6285/odata/Customers/$delta",
+  "value": [
+    {
+      "Id": 1,
+      "Name": "Customer1",
+      "Orders@delta": [
+        {
+          "@id": "Orders(1)",
+          "Amount": 14.12
+        },
+        {
+          "@id": "Orders(2)",
+          "Amount": 23.21
+        }
+      ]
+    }
+  ]
+}
+```
+
+### Sample Request 4
+This sample request demonstrates how to update the age of an existing customer and remove a link to an existing order in a bulk update request:
+
+```csharp
+var context = new Container(new Uri("http://localhost:6285/odata/"));
+
+var customer = context.Customers.FirstOrDefault(a=>a.Id == 1);
+
+customer.Age = 35;
+
+context.UpdateObject(customer);
+
+var order = context.Orders.FirstOrDefault(a=>a.Id == 2);
+
+context.DeleteLink(customer, "Orders", order);
+
+DataServiceResponse response = await context.BulkUpdateAsync(customer);
+```
+The code above generates the following bulk update request payload: 
+
+```http
+PATCH http://localhost:6285/odata/Customers
+```
+Request body:
+
+```json
+{
+  "@context": "http://localhost:6285/odata/Customers/$delta",
+  "value": [
+    {
+      "Id": 1,
+      "Name": "Customer1",
+      "Age": 56,
+      "Orders@delta": [
+        {
+          "@removed": {
+            "reason": "changed"
+          },
+          "@id": "http://localhost:6285/odata/Orders(2)"
+        }
+      ]
+    }
+  ]
+}
+```
+
+## Things to Note: 
+1. The `BulkUpdateAsync` method should be called with the top-level objects that you want to perform a bulk update on.
+
+2. There is a complementary `BulkUpdate` method applicable in synchronous scenarios.
+
+3. If the request gets processed successfully, the `DataServiceResponse` returned from the call to `BulkUpdateAsync` will have all the `ChangeOperationResponse`'s for all the operations in the bulk update request. If any of the operations failed, then the failed operation's `ChangeOperationResponse`'s descriptor's `SaveResultWasProcessed` value will be `Unchanged`. Also, the descriptor's `SaveError` property will have the error of the failed operation.
+
```

Comment:
Remove this line so that the screenshot will appear nested - same level as the text for # 3.